src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp

```diff
@@ -9,19 +9,25 @@ bool ov::intel_cpu::ACLInterpolateExecutor::init(const InterpolateAttrs &interpo
                                                  const std::vector <MemoryDescPtr> &srcDescs,
                                                  const std::vector <MemoryDescPtr> &dstDescs,
                                                  const dnnl::primitive_attr &attr) {
-    InterpolateExecutor::init(interpolateAttrs, srcDescs, dstDescs, attr);
     aclInterpolateAttrs = interpolateAttrs;
-    auto& coord_mode = aclInterpolateAttrs.coordTransMode;
-    auto& inter_mode = aclInterpolateAttrs.mode;
+    InterpolateExecutor::init(aclInterpolateAttrs, srcDescs, dstDescs, attr);
     acl_coord = arm_compute::SamplingPolicy::TOP_LEFT;
     auto& out_shape = dstDescs[0]->getShape().getDims();
-
-    if ((coord_mode == InterpolateCoordTransMode::pytorch_half_pixel && out_shape[2] > 1 && out_shape[3] > 1) ||
-        coord_mode == InterpolateCoordTransMode::half_pixel) {
+    int index_h, index_w;
+    if (dstDescs[0]->hasLayoutType(LayoutType::ncsp)) {
+        index_h = 2;
+        index_w = 3;
+    } else if (dstDescs[0]->hasLayoutType(LayoutType::nspc)) {
+        index_h = 1;
+        index_w = 2;
```

Comment:
Please create a static method to avoid duplication

Reply:
@allnes corrected